library/src/main/java/com/yayandroid/locationmanager/providers/locationprovider/GooglePlayServicesLocationProvider.java

```diff
@@ -26,8 +32,12 @@
     private final WeakReference<FallbackListener> fallbackListener;
 
     private boolean settingsDialogIsOn = false;
-    private boolean waitingForConnectionToRequestLocationUpdate = true;
-    private int suspendedConnectionIteration = 0;
+
+    /**
+     * Tracks the status of the location updates request.
+     */
+    boolean mRequestingLocationUpdates = false;
```

Comment:
Please follow the general naming convention in the library, no Hungarian Notation.

Reply:
Also, you don't really need this. Please use `setWaiting` instead, since it is used from multiple parts in the provider as well.